src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportStatusBauService.cs

```diff
@@ -48,35 +51,62 @@ public class ImportStatusBauService : IImportStatusBauService
 
                     return queryResults
                         .OrderByDescending(result => result.Timestamp)
+                        .Where(result =>
```

Comment:
Don't need a separate `Where`, would combine it with the `Select` so that you either return a `ImportStatusBau` or `null` depending on the existence of the `ReleaseFileReference`.

You can then filter the nulls out afterwards.